Doc/library/dataclasses.rst

```diff
@@ -102,36 +102,36 @@ Module contents
      If the class already defines :meth:`!__repr__`, this parameter is
      ignored.
 
-   - ``eq``: If true (the default), an :meth:`~object.__eq__` method will be
+   - *eq*: If true (the default), an :meth:`~object.__eq__` method will be
      generated.  This method compares the class as if it were a tuple
      of its fields, in order.  Both instances in the comparison must
      be of the identical type.
 
      If the class already defines :meth:`!__eq__`, this parameter is
      ignored.
 
-   - ``order``: If true (the default is ``False``), :meth:`~object.__lt__`,
+   - *order*: If true (the default is ``False``), :meth:`~object.__lt__`,
      :meth:`~object.__le__`, :meth:`~object.__gt__`, and :meth:`~object.__ge__` methods will be
      generated.  These compare the class as if it were a tuple of its
      fields, in order.  Both instances in the comparison must be of the
-     identical type.  If ``order`` is true and ``eq`` is false, a
+     identical type.  If *order* is true and *eq* is false, a
      :exc:`ValueError` is raised.
 
      If the class already defines any of :meth:`!__lt__`,
      :meth:`!__le__`, :meth:`!__gt__`, or :meth:`!__ge__`, then
      :exc:`TypeError` is raised.
 
-   - ``unsafe_hash``: If ``False`` (the default), a :meth:`~object.__hash__` method
-     is generated according to how ``eq`` and ``frozen`` are set.
+   - *unsafe_hash*: If ``False`` (the default), a :meth:`~object.__hash__` method
+     is generated according to how *eq* and *frozen* are set.
 
      :meth:`!__hash__` is used by built-in :meth:`hash()`, and when objects are
      added to hashed collections such as dictionaries and sets.  Having a
      :meth:`!__hash__` implies that instances of the class are immutable.
      Mutability is a complicated property that depends on the programmer's
      intent, the existence and behavior of :meth:`!__eq__`, and the values of
-     the ``eq`` and ``frozen`` flags in the ``@dataclass`` decorator.
+     the *eq* and *frozen* flags in the ``@dataclass`` decorator.
 
-     By default, ``@dataclass`` will not implicitly add a :meth:`~object.__hash__`
+     By default, :func:`@dataclass <dataclass>` will not implicitly add a :meth:`~object.__hash__`
```

Comment:
nit: this link also seems a bit unnecessary to me, since we're already in the docs for `@dataclass` here

Reply:
Yep, makes sense. Good catch.